packages/@azure/eventhubs/client/lib/connectionContext.ts

```diff
@@ -63,6 +63,34 @@ export namespace ConnectionContext {
     return finalUserAgent;
   }
 
+  /**
+   * Removes all the sender links , receiver links, cbs session and management session to the Event Hub.
+   * @returns {void} void
+   */
+   function remove(connectionContext: ConnectionContext): void {
+    try {
+        // Remove all the senders link.
+       for (const senderName of Object.keys(connectionContext.senders)) {
+           connectionContext.senders[senderName].remove();
```

Comment:
I would log which senders we are closing, and the same for receivers.

Reply:
The final function that is actually removing the underlying rhea link is doing the logging. Therefore, logging here is not needed